boards.txt

```diff
@@ -833,6 +833,14 @@ GenF0.menu.pnum.DEMO_F030F4_HSI.build.board=DEMO_F030F4_HSI
 GenF0.menu.pnum.DEMO_F030F4_HSI.build.product_line=STM32F030x6
 GenF0.menu.pnum.DEMO_F030F4_HSI.build.variant=DEMO_F030F4
 
+# Generic_F042Cx
+GenF0.menu.pnum.Gen_F042Cx=Generic_F042C6 (internal RC oscillator)
```

Comment:
It should be fine to provide a system clock config to test several configs as done in GenF4.
HSE, HSE bypass, HSI.

Reply:
I'm not sure how to do it